src/main/java/com/aws/greengrass/util/platforms/unix/linux/Cgroup.java

```diff
@@ -5,81 +5,117 @@
 
 package com.aws.greengrass.util.platforms.unix.linux;
 
-import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
+import com.aws.greengrass.lifecyclemanager.GreengrassService;
 
+import java.io.IOException;
 import java.nio.file.Path;
-import java.nio.file.Paths;
+import java.util.List;
 
 /**
- * Represents Linux cgroup v1 subsystems.
+ * Represents Linux cgroup subsystems.
  */
-@SuppressFBWarnings(value = "DMI_HARDCODED_ABSOLUTE_FILENAME", justification = "Cgroup virtual filesystem path "
-        + "cannot be relative")
-public enum Cgroup {
-    Memory("memory"), CPU("cpu,cpuacct"), Freezer("freezer", "freezer");
+public class Cgroup {
```

Comment:
Why does this class exist at all?

All of its methods are just calling into the same methods in `CGroupSubSystemPath`, so why does this class exist at all? Just use `CGroupSubSystemPath` directly if that's what we need.